extensions/2.0/Khronos/KHR_materials_diffuse_transmission/README.md

```diff
@@ -0,0 +1,252 @@
+| **diffuseTransmissionFactor**       | `number`                                                            | The percentage of reflected, non-specularly reflected light that is transmitted through the surface via the Lambertian diffuse transmission, i.e., the strength of the diffuse transmission effect. | No, default: `0`         |
+| **diffuseTransmissionTexture**      | [`textureInfo`](/specification/2.0/README.md#reference-textureInfo) | A texture that defines the strength of the diffuse transmission effect, stored in the alpha (A) channel. Will be multiplied by the diffuseTransmissionFactor.                                       | No                       |
+| **diffuseTransmissionColorFactor**  | `number[3]`                                                         | The color of the transmitted light.                                                                                                                                                                 | No, default: `[1, 1, 1]` |
+| **diffuseTransmissionColorTexture** | [`textureInfo`](/specification/2.0/README.md#reference-textureInfo) | A texture that defines the color of the transmitted light, stored in the RGB channels and encoded in sRGB. This texture will be multiplied by diffuseTransmissionColorFactor.                       | No                       |
+
+
+The strength of the effect is controlled by `diffuseTransmissionFactor` and `diffuseTransmissionTexture`, combined via multiplication to describe a single value.
+```
+diffuseTransmission = diffuseTransmissionFactor * diffuseTransmissionTexture.a
+```
+The color of the effect is controlled by `diffuseTransmissionColorFactor` and `diffuseTransmissionColorTexture`, combined via multiplication to describe a single color.
+```
+diffuseTransmissionColor = diffuseTransmissionColorFactor * diffuseTransmissionColorTexture.rgb
+```
+## Examples
+
+The examples use `diffuseTransmissionColorTexture` from `baseColorTexture`.
+|                No transmission                 | <sub>`diffuseTransmissionFactor: : 0.25`</sub> | <sub>`diffuseTransmissionFactor: 0.25`<br>`diffuseTransmissionColorFactor: [1.0,0.9,0.85]`</sub> |
+| :--------------------------------------------: | :--------------------------------------------: | :----------------------------------------------------------------------------------------------: |
+| ![](figures/teatime_backlit_no_trans.jpg) <br> |        ![](figures/teatime_backlit.jpg)        |                             ![](figures/teatime_backlit_colored.jpg)                             |
+|  ![](figures/teatime_side_no_trans.jpg) <br>   |         ![](figures/teatime_side.jpg)          |                              ![](figures/teatime_side_colored.jpg)                               |
```

Comment:
Minor cleanup — I think if this table were transposed, the images in the same column would have the same size and differences would be easier to compare:

| Case | Backlit | Side |
|------|---------|------|
| No Transmission | ![](https://placekitten.com/g/200/200) | ![](https://placekitten.com/g/200/200) |
| <sub>`diffuseTransmissionFactor: 0.25`</sub> | ![](https://placekitten.com/g/200/200) | ![](https://placekitten.com/g/200/200) |
| <sub>`diffuseTransmissionFactor: 0.25`<br>`diffuseTransmissionColorFactor: [1.0,0.9,0.85]`</sub> | ![](https://placekitten.com/g/200/200) | ![](https://placekitten.com/g/200/200) |